akka-http-core/src/main/scala/akka/http/impl/util/package.scala

```diff
@@ -127,6 +127,66 @@ package object util {
 
 package util {
 
+  import akka.http.scaladsl.model.{ ContentType, HttpEntity }
```

Comment:
I don't mind the package objects as they are in Http currently but if you have feelings about them I'll open a ticket and re-visit this :) https://github.com/akka/akka/issues/18477